hackathon_site/event/jinja2/event/landing.html

```diff
@@ -12,299 +14,365 @@
-                <a href="https://www.google.com/" class="col s4 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <a href="https://www.google.com/" class="col s4 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <br />
-                <!-- Silver -->
-                <a href="https://www.google.com/" class="col s3 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <a href="https://www.google.com/" class="col s3 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <a href="https://www.google.com/" class="col s3 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <a href="https://www.google.com/" class="col s3 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <br />
-                <!-- Bronze -->
-                <a href="https://www.google.com/" class="col s2 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <a href="https://www.google.com/" class="col s2 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <a href="https://www.google.com/" class="col s2 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <a href="https://www.google.com/" class="col s2 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <a href="https://www.google.com/" class="col s2 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <a href="https://www.google.com/" class="col s2 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
+                <img src="{{ static('event/images/sponsors.svg') }}" alt="One big sponsor image"/>
                 <br />
             </div>
             <p class="center">
                 If you would like to sponsor us, feel free to reach out:
-                <a href="mailto:hello@email.com" class="hoverLink">hello@email.com</a>
+                <a href=mailto:{{ contact_email }}" class="hoverLink">{{ contact_email }}</a>
```

Comment:
Created an issue to update this in the template repo: https://github.com/ieeeuoft/hackathon-template/issues/240